deploy/docker/common/nginx-config.conf

```diff
@@ -11,6 +11,16 @@ server {
     gzip_buffers 16 8k;
     gzip_http_version 1.1;
 
+    # to handle uri issue 414 from nginx
+    fastcgi_buffers 8 16k;
+    fastcgi_buffer_size 32k;
+
+    client_max_body_size 24M;
+    client_body_buffer_size 128k;
+
+    client_header_buffer_size 5120k;
+    large_client_header_buffers 16 5120k;
```

Comment:
For most requests, default buffer of 1K bytes is enough.
So, I suppose we shouldn't overwrite `client_header_buffer_size`.

When that buffer size is exceeded, larger buffers set by `large_client_header_buffers` are allocated.
Which is what we require to resolve 414 issue.

default value of `large_client_header_buffers` is `4 8k` which is maximum number and size of buffers.
`16 5120k` is  quite a lot. Most solutions I see out there are `4 16k` or `4 32k`.

@palashgdev could you please try with those instead?

Reply:
yes `8 16k` is working

Reply:
Awesome!